### ISSUE TITLE
Image recognition of score screenshot for automatic verification (LMAO)

### ISSUE THREAD

**Issue by ifSomeday** [ifSomeday/FlagBot#1]
Pretrained accuracy very low
Need to finetune EAST/tesseract for this.
Use Arial size 10 font
